changelog.d/gelf_at_character.enhancement.md

```diff
@@ -0,0 +1 @@
+gracefully accept @ characters in labels when decoding GELF
```

Comment:
The GH workflow to validate the changelog fragment should have flagged this as not having the "authors" bit at the end. I found the bug and will submit a PR for that.

```suggestion
Gracefully accept `@` characters in field names when decoding GELF.
authors: MartinEmrich
```